x/stake/handler.go

```diff
@@ -49,6 +49,19 @@ func NewEndBlocker(k Keeper) sdk.EndBlocker {
 
 //_____________________________________________________________________
 
+// InitGenesis - store genesis parameters
+func InitGenesis(ctx sdk.Context, k Keeper, data json.RawMessage) error {
```

Comment:
We don't need to take `json.RawMessage` anymore - see https://github.com/cosmos/cosmos-sdk/blob/develop/examples/democoin/app/app.go#L145 - instead specify a full `GenesisState` struct for Gaia.

Reply:
addressed